docs/development/plugins-job-types.md

```diff
@@ -68,6 +68,9 @@ Every wrapper has to follow some rules:
 - You MAY implement [swagger](https://swagger.io/tools/swagger-ui/) documentation for your endpoints on root endpoint.
 - You MAY implement `/metrics` endpoint for exposing [Prometheus](https://prometheus.io/) metrics.
 - You MAY expose any other endpoints.
+- Chain calls to other jobs SHOULD be made by
```

Comment:
Is it obvious to an outside reader what a "chain call" is? I find it quite intuitive, but maybe something like this is more understandable to those not in the know
```suggestion
- Calls from jobs to other jobs SHOULD be made by
```